src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
The function name 'makeCalculator' is appropriate as it clearly describes the purpose of the function. Good job following the naming convention from the checklist.